Sources/Sentry/SentrySDK.m

```diff
@@ -632,6 +632,25 @@ + (void)stopProfiler
 
     [SentryContinuousProfiler stop];
 }
+
++ (void)startProfileSession
+{
+    // TODO: log a debug message and bail if the profiling session is sampled
+    // with respect SentryOptions.profileSessionSampleRate and the profiler is already running
+
+    // TODO: log a debug message and bail if the profile session is not sampled
+
+    // TODO: log a debug message and bail if SentryOptions.profileLifecycle is set to trace
+
+    [SentryContinuousProfiler start];
```

Comment:
`m`: It would be great if we could write a simple test validating that this method actually starts the SentryContinuousProfiler.

Reply:
If you mean specifically for the internal method `+ [SentryContinuousProfiler start]`, we do have a test for that already: https://github.com/getsentry/sentry-cocoa/blob/c964075f766455bae09be6d409d8823eb75a62ed/Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift#L31-L33 

the actual assertion is made at https://github.com/getsentry/sentry-cocoa/blob/c964075f766455bae09be6d409d8823eb75a62ed/Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift#L176-L179

Reply:
but i will add more tests for the public API entrypoint of SentrySDK.startProfileSession to exercise all the early exits there for the various checks, as i did in https://github.com/getsentry/sentry-cocoa/pull/4961